src/features/metadata-based-view/__tests__/IconWithTooltip.test.js

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import { IconWithTooltipComponent as IconWithTooltip } from '../IconWithTooltip';
+
+import messages from '../../../elements/common/messages';
+import { CANCEL_ICON_TYPE, EDIT_ICON_TYPE, SAVE_ICON_TYPE } from '../constants';
+
+describe('features/metadata-based-view/IconWithTooltip', () => {
+    const getWrapper = props => mount(<IconWithTooltip {...props} />);
```

Comment:
Since these tests aren't dependent on component lifecycle methods, can we use `shallow` here instead?